WebApp/autoreduce_webapp/selenium_tests/tests/test_help_page.py

```diff
@@ -0,0 +1,24 @@
+from selenium_tests.pages.help_page import HelpPage
+from selenium_tests.tests.base_tests import NavbarTestMixin, BaseTestCase, \
+    FooterTestMixin
+from selenium.webdriver.support.wait import WebDriverWait
+
+
+class TestHelpPage(NavbarTestMixin, BaseTestCase, FooterTestMixin):
```

Comment:
It would be nice to have some extra tests, perhaps ones that enter some text in the search and assert that the right boxes remain shown?

This helper for writing to a field should make it easier to do that:https://github.com/ISISScientificComputing/autoreduce/blob/699-improve-help-page-search/WebApp/autoreduce_webapp/selenium_tests/pages/component_mixins/rerun_form_mixin.py#L12-L15

Reply:
I'm planning on adding these tests:
```py
# def test_all_topics_have_headers
# def test_all_topics_have_a_category
# def test_all_topics_have_content
# def test_filter_by_categories
# def test_scroll_to_header
```